samples/cellular/nrf_cloud_multi_service/Kconfig

```diff
@@ -295,7 +295,7 @@ config COAP_SHADOW_CHECK_RATE_SECONDS
 
 config COAP_SHADOW_THREAD_STACK_SIZE
 	int "CoAP Shadow Thread Stack Size (bytes)"
-	default 2048
```

Comment:
commit log says enable PSM by default, but I don't see it anywhere, possibly left out by mistake?

Reply:
Also, is this the PR where you were going to remove the redundant PSM KConfig?

Reply:
Aha, I see you've done this in https://github.com/nrfconnect/sdk-nrf/pull/12492 instead -- so then I think that mention just needs to be removed from the commit log

Reply:
Good catch -- I removed the change I had made to the Kconfig to default to y (which did nothing, as we discussed) but forgot to remove the comment.